app/utils/prepare.js

```diff
@@ -13,12 +13,13 @@ type PrepareFn = (props: Object, dispatch: Dispatch<*>) => Promise<*>;
  */
 export default function prepare(
   prepareFn: PrepareFn,
-  watchProps: ?Array<string>
+  watchProps?: Array<string> = []
 ) {
   // Returns true if any of the given watchProps have changed:
   const componentWillReceiveProps = (oldProps, newProps) =>
-    watchProps &&
-    watchProps.some(key => get(oldProps, key) !== get(newProps, key));
+    watchProps
+      .concat('loggedIn')
+      .some(key => get(oldProps, key) !== get(newProps, key));
```

Comment:
Won't this crash if `watchProps` is undefined?

<img width="497" alt="skjermbilde 2017-10-30 kl 15 20 52" src="https://user-images.githubusercontent.com/14221386/32175691-f0f735f6-bd85-11e7-8df8-f5f90f22084d.png">